app/models/search/tag_query.rb

```diff
@@ -54,7 +54,7 @@ def sort
       column = "name.keyword"
       direction ||= "asc"
     end
-    sort_hash = { column => { order: direction } }
+    sort_hash = { column => { order: direction }, "id" => { order: direction } }
```

Comment:
> I didn't find where we search and sort for tags, but I think it wouldn't hurt

We can't sort tags from the UI yet, but that's a planned feature (#4144) so let's definitely add this.